index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cmart | Login</title>
+    <link
+      rel="stylesheet"
+      href="https://aceeui.netlify.app/stylesheet/final.style.css"
+    />
+    <link rel="stylesheet" href="/styles/homepage.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="#" class="logo_link">
+          <i class="fas fa-birthday-cake fa-lg cake-logo"></i>
+        </a>
+        <h1 class="heading-title">Cmart</h1>
+      </div>
+      <ul class="list"></ul>
+      <ul class="list_end">
+        <li class="list_item login">
+          <div class="login-icon">
+            <a class="list_item--link" href="./markups/login.html">
+              <i class="fas fa-user"></i>
+              <span class="login-txt">Login</span>
+            </a>
+          </div>
+        </li>
+        <li class="list_item">
+          <a class="list_item--link" href="https://twitter.com/jn_dubey">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </li>
+        <li class="list_item">
+          <a
+            class="list_item--link"
+            href="https://www.linkedin.com/in/aditya-dubey-47ba6870/"
+          >
+            <i class="fab fa-linkedin"></i>
+          </a>
+        </li>
+        <li class="list_item">
+          <a
+            class="list_item--link"
+            href="https://github.com/dubeyaditya29/Cmart"
+          >
+            <i class="fab fa-github"></i>
+          </a>
+        </li>
```

Comment:
These links are there in both header and footer. Since it is a E-commerce app, it would be better to have them in footer.

Reply:
Sure thanks for the suggestion.